proxy/src/http/org.rs

```diff
@@ -74,6 +77,31 @@ fn get_filter(
         .and_then(handler::get)
 }
 
+/// `GET /orgs`
```

Comment:
Thought about this a bit more. Besides that it is per spec not expected for a server to parse any body for GET request, I think this endpoint should be modeled a bit differently. First I thought along listing with a `by` constraints, i.e. member id. Which is usually expressed with query parameters. Which then begs the question what if the parameter is not passed, multiple permutations are passed for different constraints and so on.

What we actually want is the list of a orgs for a specific user. That user must exist, therefore I think a better semantic would be to have this endpoint in `user.rs`, under `/users/<id>/orgs` and make it unambiguous and easy to use. This will clarify semantics for it, won't require any extensive or complicated documentation on how to satisify the use-case.

Reply:
👍 Moved!